.github/workflows/image-push.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read  
+
+jobs:
+  push-multiarch:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Login to Container Registry
+      uses: docker/login-action@v3
+      with:
+        username: ${{ secrets.docker_username }}
+        password: ${{ secrets.docker_password }}
```

Comment:
According to EIO, the secrets must be read in a different way ==> https://github.com/rancher/image-build-addon-resizer/blob/main/.github/workflows/image-push.yml#L22-L33